app/src/main/java/org/wikipedia/descriptions/DescriptionEditActivity.java

```diff
@@ -69,31 +72,22 @@ public void onDescriptionEditSuccess() {
 
     @Override
     public void onBottomBarContainerClicked(@NonNull InvokeSource invokeSource) {
```

Comment:
This change is not needed, as the feed card makes this a switch statement. Could we hold this off till that gets merged. So if you can remove this change here, rest of it looks good to go

Reply:
The idea is to reduce the unnecessary value been created and also reduce the repeated functions get called.

I think it is easy that just add `FEED_CARD_SUGGESTED_EDITS_TRANSLATE_DESC`, `FEED_CARD_SUGGESTED_EDITS_TRANSLATE_IMAGE_CAPTION` to line `76`

and add `FEED_CARD_SUGGESTED_EDITS_IMAGE_CAPTION` and `FEED_CARD_SUGGESTED_EDITS_TRANSLATE_IMAGE_CAPTION` to line `82`

Reply:
Yes @cooltey  makes sense for now... the switch was to facilitate readability if we end up having more types in the future... but this is more efficient for now... will merge this!